permafrost/tests.py

```diff
@@ -505,6 +506,19 @@ def test_role_update_POST_updates_name(self):
         updated_role = PermafrostRole.objects.get(pk=self.pf_role.pk)
         self.assertEqual(updated_role.name, "Test Change")
     
+    def test_role_update_POST_updates_when_no_values_are_changed(self):
+        uri = reverse('permafrost:role-update', kwargs={'slug': 'test-role'})
```

Comment:
This test triggers the integrity errror before I added the get_form_kwargs method below